2017ClubChampionshipEvent1.htm

```diff
@@ -270,13 +194,13 @@
  5304;width:112pt'>
```

Comment:
Line 265 (189) is the title itself, and it's unchanged (hence not being able to add this comment in quite the right place).

The html looks fine, so I think we might still have the funny characters instead of the spaces.